config/samples/mongodb.com_v1_mongodbcommunity_cr.yaml

```diff
@@ -23,6 +23,9 @@ spec:
       scramCredentialsSecretName: my-scram
   additionalMongodConfig:
     storage.wiredTiger.engineConfig.journalCompressor: zlib
+  statefulSet:
+    spec:
+      serviceName: newdatabasename
```

Comment:
I don't think we need to push this change, can you please revert it?

Reply:
Oops sorry, I thought I had made sure not to push it :)